packages/runtime/src/shared/types.ts

```diff
@@ -13,6 +13,7 @@ export interface EnsembleWidgetProps<
 > {
   id?: string;
   styles?: T;
+  htmlAttributes?: Record<string, unknown>;
```

Comment:
Value type is Expression<string>

Reply:
@evshi here htmlAttributes will be the object with the key value pair. so is this right ?
`htmlAttributes?: Record<string, Expression<string>>`